src/clj/rems/css/styles.clj

```diff
@@ -563,6 +563,10 @@
    [:.navbar-toggler {:border-color (get-theme-attribute :color1)}]
    [:.nav-link
     :.btn-link
+    ;; TODO this use of link-color is preventing us from moving the
+    ;; default for link-color into config-defaults.edn. At least
+    ;; theme-findata has set :link-color (to something different
+    ;; than :color3) and not set :nav-color.
     {:color (get-theme-attribute :nav-color :link-color :color3)
```

Comment:
Is nav-color relatively new so that's why it isn't set? I think link colors in general are either blue or one of the brand colors.

Reply:
I guess theme authors set `:link-color` and then they're glad that the navbar got "fixed" too.

The practical situation is this:
- theme-csc sets nav-color and link-color
- The other production themes like thl, lbr and findata set link-color and don't set nav-color

So actually nobody is using the callback to color3, so I could change this to `(get-theme-attribute :nav-color :link-color)` and add the default for `:link-color`!

Reply:
I implemented this